e2e_tests/tests/fixtures/generic_task/run.py

```diff
@@ -0,0 +1,9 @@
+import time
+
+t = 5
+print("start")
+for i in range(t):
+    if i % 10 == 0:
+        print("Working...")
+    time.sleep(1)
+print("end")
```

Comment:
It looks like `run.py` runs in a few places, and takes 5 seconds to run.

Is there a reason why it can't be instant?  When it comes to automated tests, 0 seconds is way better than 5 seconds (or 1 second better than 6, more likely).

Reply:
Reduced to 1 sec. Originally some time was created to that for manual testing we could pause/unpause or do a restore. For future manual testing I think someone can just edit the file to increase the time.

Reply:
Can you reduce to zero seconds, and use a different fixture for the pause/unpause test specifically?  Or just an inline command like `entrypoint: ["sleep", "5"]`?

I feel very strongly about leaving unnecessary sleeps anywhere in automated testing.  Seconds add up, and old fixtures have a habit of getting reused over and over in new tests. 

(side note, please don't mark the comment as resolved until I have had a chance to see what the outcome was, unless it was something super trivial like correcting a typo)

Reply:
added new fixture for pause and removed sleep from the other